docs/open-source/_toc.json

```diff
@@ -0,0 +1,44 @@
+{
+  "title": "Open source",
```

Comment:
I'm not sure "Open source" is the correct title grammatically. I think it should be "Open Source" or "Open-source resources"

Reply:
I have no idea. @abbycross @beckykd thoughts?

Reply:
> I think it should be "Open Source"

We always use sentence case

Reply:
I vote for `Open-source resources`